owl/EASE-PROC.owl

```diff
@@ -149,95 +217,6 @@ As such, this concept can be applied only to motions involving some PhysicalAgen
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-PROC.owl#HeadTurning -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-PROC.owl#HeadTurning">
```

Comment:
these were all just moved by protege. @mpomarlan please use protege to save the files to avoid this